test/jdk/java/beans/XMLEncoder/Test4631471.java

```diff
@@ -25,7 +25,7 @@
  * @test
  * @bug 4631471 6972468
  * @summary Tests DefaultTreeModel encoding
- * @run main/othervm -Djava.security.manager=allow Test4631471
+ * @run main/othervm Test4631471
```

Comment:
Other tests in the set removed `/othervm`. Is it left intentionally?

Looks like this question applies to all tests in `XMLEncoder/` directory.